server/cmwell-ws/app/controllers/Health.scala

```diff
@@ -119,9 +116,10 @@ class Health @Inject()(crudServiceFS: CRUDServiceFS, ws: WSClient) extends Injec
 
   def getKafkaStatus = Action.async {implicit req =>
     val javaHomeLocation = s"$path/../java"
-    val javaHomeAddition = if (Files.exists(Paths.get(javaHomeLocation)))
-      s"JAVA_HOME=$path/../java"
-    else ""
+
+    val javaHomeAddition = s"""if [ -d $javaHomeLocation ]
+        then JAVA_HOME=$javaHomeLocation ;
```

Comment:
this won't work and you will need export (even though that without the "if" it will work)
(at least without the bash -c it doesn't work)